include/swift/Serialization/ModuleDependencyScanner.h

```diff
@@ -41,18 +42,22 @@ namespace swift {
 
       /// Location where pre-built moduels are to be built into.
       std::string moduleCachePath;
+
+      Optional<SwiftDependencyTracker> dependencyTracker;
     public:
       Optional<ModuleDependencyInfo> dependencies;
 
       ModuleDependencyScanner(ASTContext &ctx, ModuleLoadingMode LoadMode,
                               Identifier moduleName,
                               InterfaceSubContextDelegate &astDelegate,
-                              ScannerKind kind = MDS_plain)
+                              ScannerKind kind = MDS_plain,
```

Comment:
Hmm, not a comment on this PR at all, but I ought to go rip this out altogether...